youtube_dl_gui/locale/es_ES/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,440 @@
+
+#: optionsframe.py:740
+msgid "mid"
+msgstr "media"
+
+#: optionsframe.py:743
+msgid "Convert to Audio"
+msgstr "Convertir a Audio"
+
+#: optionsframe.py:744
+msgid "Keep Video"
+msgstr "Mantener video"
+
+#: optionsframe.py:745
+msgid "Audio Format"
+msgstr "Formato de Audio"
+
+#: optionsframe.py:746
+msgid "Audio Quality"
+msgstr "Calidad del Audio"
+
+#: optionsframe.py:863
+msgid "default"
+msgstr "defecto"
+
+#: optionsframe.py:864
+msgid "none"
+msgstr "ninguno"
+
+#: optionsframe.py:868
+msgid "Video Format"
+msgstr "Formato del videio"
+
+#: optionsframe.py:869
+msgid "Mix Format"
+msgstr "Mezclar formato"
+
+#: optionsframe.py:928
+msgid "Restrict filenames (ASCII)"
+msgstr "Restringir los nombres de archivo (ASCII)"
+
+#: optionsframe.py:929
+msgid "ID as Name"
+msgstr "ID como nombre"
+
+#: optionsframe.py:930
+msgid "Title as Name"
+msgstr "Titulo como nombre"
+
+#: optionsframe.py:931
+msgid "Custom Template (youtube-dl)"
+msgstr "Plantilla Personalizada (youtube-dl)"
+
+#: optionsframe.py:1025
+msgid "Ignore Errors"
+msgstr "Ignorar errores"
+
+#: optionsframe.py:1026
+msgid "Open destination folder"
+msgstr "Abrir carpeta de destino"
+
+#: optionsframe.py:1027
+msgid "Write info to (.json) file"
+msgstr "Escribir información a (.json) archivo"
+
+#: optionsframe.py:1028
+msgid "Write description to file"
+msgstr "Escribir descripción a archivo"
+
+#: optionsframe.py:1029
+msgid "Write thumbnail to disk"
+msgstr "Escribir miniatura en el disco"
+
+#: optionsframe.py:1030
+msgid "Filesize"
+msgstr "Tamaño del archivo"
+
+#: optionsframe.py:1031
+msgid "Min"
+msgstr "Min"
+
+#: optionsframe.py:1032
+msgid "Max"
+msgstr "Max"
+
+#: optionsframe.py:1145
+msgid "English"
+msgstr "Ingles"
+
+#: optionsframe.py:1146
+msgid "Greek"
+msgstr "Griego"
+
+#: optionsframe.py:1147
+msgid "Portuguese"
+msgstr "Portugues"
+
+#: optionsframe.py:1148
+msgid "French"
+msgstr "Frances"
+
+#: optionsframe.py:1149
+msgid "Italian"
+msgstr "Italiano"
+
+#: optionsframe.py:1150
+msgid "Russian"
+msgstr "Ruso"
+
+#: optionsframe.py:1151
+msgid "Spanish"
+msgstr "Spañol"
+
+#: optionsframe.py:1152
+msgid "German"
+msgstr "Aleman"
+
+#: optionsframe.py:1155
+msgid "Download subtitle file by language"
+msgstr "Descargar archivo de subtítulos por idiomas"
+
+#: optionsframe.py:1156
+msgid "Download all available subtitles"
+msgstr "Descargar todos los subtítulos disponibles"
+
+#: optionsframe.py:1157
+msgid "Download automatic subtitle file (YOUTUBE ONLY)"
+msgstr "Descargar automáticamente archivos de subtitulo (SOLO YOUTUBE)"
+
+#: optionsframe.py:1158
+msgid "Embed subtitles in the video (only for mp4 videos)"
+msgstr "Integrar los subtítulos en el vídeo (solo para vídeos mp4)"
+
+#: optionsframe.py:1159
+msgid "Subtitles Language"
+msgstr "Idioma subtitulos"
+
+#: optionsframe.py:1253
+msgid "About"
+msgstr "Acerca"
+
+#: optionsframe.py:1254
+msgid "Open"
+msgstr "Abrir"
+
+#: optionsframe.py:1255
+msgid "Reset Options"
+msgstr "Restablecer Opciones"
```

Comment:
The translated string doesn't fit in the corresponding button.
It's up to you if you want to translate it to something else in order to fit or leave it as is.
Maybe you could translate it to reset or something else i don't know.